client/list.go

```diff
@@ -35,3 +46,29 @@ func (client *Client) listLowLevel(path string) (result []string, err error) {
 
 	return result, err
 }
+
+func (client *Client) listAllLowLevel(path string, subResult* []string) (result[]string, err error) {
```

Comment:
Would the `traverse` function do the same? https://github.com/fishi0x01/vsh/blob/master/client/traverse.go#L16

Reply:
Kind of will look into how to use this function for the list all command then.

Reply:
Don't reinvent the wheel here.